core/src/main/java/org/elasticsearch/index/search/SimpleQueryStringQueryParser.java

```diff
@@ -29,35 +28,61 @@
 import org.apache.lucene.search.BoostQuery;
 import org.apache.lucene.search.DisjunctionMaxQuery;
 import org.apache.lucene.search.FuzzyQuery;
+import org.apache.lucene.search.MatchNoDocsQuery;
 import org.apache.lucene.search.PrefixQuery;
 import org.apache.lucene.search.Query;
 import org.apache.lucene.search.SynonymQuery;
 import org.apache.lucene.util.BytesRef;
-import org.elasticsearch.index.analysis.ShingleTokenFilterFactory;
+import org.elasticsearch.common.unit.Fuzziness;
 import org.elasticsearch.index.mapper.MappedFieldType;
+import org.elasticsearch.index.query.AbstractQueryBuilder;
+import org.elasticsearch.index.query.MultiMatchQueryBuilder;
+import org.elasticsearch.index.query.QueryShardContext;
+import org.elasticsearch.index.query.SimpleQueryStringBuilder;
 
 import java.io.IOException;
-import java.util.Iterator;
 import java.util.Map;
 import java.util.Objects;
 import java.util.List;
 import java.util.ArrayList;
 
+import static org.elasticsearch.common.lucene.search.Queries.newUnmappedFieldQuery;
+
 /**
- * Wrapper class for Lucene's SimpleQueryParser that allows us to redefine
+ * Wrapper class for Lucene's SimpleQueryStringQueryParser that allows us to redefine
  * different types of queries.
  */
-public class SimpleQueryParser extends org.apache.lucene.queryparser.simple.SimpleQueryParser {
+public class SimpleQueryStringQueryParser extends org.apache.lucene.queryparser.simple.SimpleQueryParser {
```

Comment:
I think this no longer needs to be fully qualified since the class name is not the same any more